src/main/java/com/syncstream_dev/syncstream/model/ChatStream.java

```diff
@@ -0,0 +1,37 @@
+package com.syncstream_dev.syncstream.model;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import com.syncstream_dev.syncstream.model.message.Message;
+import com.syncstream_dev.syncstream.model.message.UserMessage;
+import com.syncstream_dev.syncstream.model.user.User;
+
+public class ChatStream {
+    private HashMap<String, Message> messages;      // Message id to Message mapping
+    private ArrayList<User> participants;
+
+    public ChatStream() {
+        messages = new HashMap<String, Message>();
+        participants = new ArrayList<User>();
```

Comment:
Java 17 can infer types: `new ArrayList<User>()` can just be `new ArrayList<>()`

We can keep it for the sake of readability.

Reply:
612afa7f2a75cb4104ee3605adf069177b233d09 Addresses this